### DISCUSSION TITLE
Custom Achievement badges from Open Source repositories

### DISCUSSION THREAD

**Discussion by kuberaspeaking** [community/community#21346] [Other Feature Feedback, Questions, & Ideas]
I would be great if open source orgs could create their custom batch and give it to people who contribute to their code. 
So someone who lets say contributed to Apache Superset will have the apache superset logo as an achievement in their profile.

**Comment by jfullstackdev** [comment]
imagine if all Orgs will have at least one badge 

![image](https://user-images.githubusercontent.com/47092464/181179712-fc3a9628-ba71-4560-aa52-67f90250584a.png)

there will be 4 million badges being issued, that would be a really tremendous amount of badges

well, any Org can actually create that but outside GitHub, I think, that's the most practical one
